incubator/dokuwiki/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
+        - name: DOKUWIKI_USERNAME
+          value: {{ default "" .Values.dokuwikiUsername | quote }}
+        - name: DOKUWIKI_FULLNAME
+          value: {{ default "" .Values.dokuwikiFullname | quote }}
+        - name: DOKUWIKI_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "fullname" . }}
+              key: dokuwiki-password
+        - name: DOKUWIKI_EMAIL
+          value: {{ default "" .Values.dokuwikiEmail | quote }}
+        - name: DOKUWIKI_WIKINAME
```

Comment:
Like the previous comment this env variable should have been named `DOKUWIKI_WIKI_NAME`